app/assets/stylesheets/v3/layout/variables.scss

```diff
@@ -107,6 +107,8 @@ $base-padding: 1rem;
 $base-border: 1px solid $stone;
 // Radius
 $base-radius: 9px;
+// Drop-shadow
+$base-box-shadow: 2.5px 4.33px 6.86px 0.14px rgba(24, 24, 24, 0.35)!important;
```

Comment:
does this need to be !important? I had added this variable on my branch and I didn't need an !important on it.

Reply:
It does! There is another class specifically for this div's transition that says not to apply a drop shadow.
```
.off-canvas.is-transition-overlap.is-open {
    box-shadow: none;
}
```
I believe this is a foundation class that I needed to override.

Reply:
It does! There is another class specifically for this div's transition that takes priority and says not to apply a drop shadow.
```
.off-canvas.is-transition-overlap.is-open {
    box-shadow: none;
}```
I believe this is a foundation class that I needed to override.

Reply:
okay! 👍